.github/workflows/build-docker-onnx-subgr.yml

```diff
@@ -0,0 +1,28 @@
+name: Build docker image for tools/onnx_subgraph CI
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build-docker-image:
+    if: github.repository_owner == 'Samsung'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version: [ 'u2204' ]
+    steps:
+      - name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Install Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Build Docker Image
+        uses: docker/build-push-action@v6
+        with:
+          file: ./onnx_subgraph/docker/Dockerfile.${{ matrix.version }}
+          push: true
+          tags: nnfw/onnx-subgraph-build-${{ matrix.version }}
```

Comment:
Will you use different image name for each ubuntu version? (not using tag?)

Reply:
I prefer to use tag because each image becomes repository on docker hub account. 
If our `nnfw` account have only `onnx-subgraph-build-` series repository, it's OK. But `nnfw` account also have image repository for `onert` and `circle-mlir`.

Anyway, it's OK to use this naming policy now because currently only one image is used. But if we need to support more ubuntu version at once, I think we should change naming policy to use tag.

Reply:
Thanks for your opinion. I'll post an issue for this as I have plans to support also U20.04 and maybe U24.04.